tests/Unit/Helpers/Evolution/Systems/Cce/AnalyticSolutions/AnalyticDataHelpers.hpp

```diff
@@ -96,19 +96,20 @@ struct SphericalSolutionWrapper : public SphericalSolution {
         python_file, "news", sin_theta, time_vector,
         Scalar<DataVector>{DataVector{size, args}}...);
 
+    Approx custom_approx = Approx::custom().epsilon(1.e-11).scale(1.0);
```

Comment:
Perhaps take the tolerance as an argument to `test_spherical_metric` function, since the precision is problem-dependent? (If you have trouble with the `Args` parameter pack, you could take it as a `std::tuple<Args...>`) Up to you if you think this is worth the time.